alembic/versions/21_add_vectors_schema_and_article_chunks.py

```
@@ -0,0 +1,99 @@
+    op.execute("CREATE EXTENSION IF NOT EXISTS vector")
+
+    # Parent table — stores article metadata for search result joins
+    op.create_table(
+        "articles",
+        sa.Column("id", UUID(as_uuid=True), primary_key=True),
+        sa.Column("url", sa.Text(), nullable=False, unique=True),
+        sa.Column("title", sa.Text(), nullable=True),
+        sa.Column("source", sa.Text(), nullable=True),
+        sa.Column("public_article_id", UUID(as_uuid=True), nullable=True),
+        sa.Column("topic_id", UUID(as_uuid=True), nullable=True),
+        sa.Column("metadata", JSONB, nullable=True),
```

Ruff (0.15.17)

[error] Undefined name `JSONB`

(F821)